priority/management/commands/calculate-priority.py

```diff
@@ -0,0 +1,121 @@
+from django.core.management.base import BaseCommand, CommandError
+from sqlalchemy import create_engine, Table
+import numpy as np
+import pandas as pd
+from datetime import date, timedelta
+from project.models import Project
+from funding.models import Attribution
+from priority.models import slurm_priority
+import sys
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        #number of QOS levels
+        QOS_levels=4.0
+        Today=date.today()
+        #engine for reading/writing sql files
+        # sqlite://<nohostname>/<pathtofile>
+        engine = create_engine('sqlite:///db.sqlite3', echo=False)
+        df = pd.read_csv ('~/Documents/full_sacct_dump.csv',sep='|')
+        #df = pd.read_csv ('~/Documents/Cogs3/cogs3/priority/fixtures/TEST_dump.csv',sep='|')
+        #remove NANs
+        df=df.dropna(how='any')
+        #convert cpu time from core seconds to core hours
+        df.loc['CPUTimeRAW']=df['CPUTimeRAW'].div(3600)
+        #df['CPU-HOURS']=cpuhours.astype(int)
+        #seperate into gpu and not gpu
+        gpu=df[(df.Partition=="gpu")|(df.Partition=="xgpu")]
+        compute=df[(df.Partition!="gpu")|(df.Partition!="xgpu")]
+        compute.to_csv(path_or_buf='~/Documents/Cogs3/cogs3/compute.csv')
```

Comment:
is this a test dump? If so, it should be optional, with the path given as an optional argument